scope_test.go

```diff
@@ -313,6 +314,43 @@ func TestWriteReportLoop(t *testing.T) {
 	r.WaitAll()
 }
 
+func TestInstrumentedCallSuccess(t *testing.T) {
+	r := newTestStatsReporter()
+	s, closer := NewRootScope(ScopeOptions{Reporter: r}, 10)
+	defer closer.Close()
+
+	r.cg.Add(1)
+	r.tg.Add(1)
+	err := NewInstrumentedCall(s, "testInstrumented").Exec(func() error {
```

Comment:
nit: For consistency can you name these as underscore separated rather than lower camel case?  This matches the rest of the tests and examples in the project. i.e. `test_instrumented`, etc.